generator/consts.py

```diff
@@ -34,13 +34,15 @@
 iso_639_languages = {
     "BG": "Bulgarian",
     "BS": "Bosnian",
-    "CS": "Czech",
+    "CA": "Catalan",
+    "CZ": "Czech",
```

Comment:
We are using the ISO-639-1 codes here.
[Czech](https://de.wikipedia.org/wiki/Tschechische_Sprache) has the code `CS`.

If our data is wrong in the csv files we should correct that.